### ISSUE TITLE
Bug: Nicotin base calculation

### ISSUE THREAD

**Issue by giacomo-pati** [skydel0/Liquido#2]
Calculation on a mixture between volume (ml) and weight (g) seems wrong. 
I. E. I have a Nic base of 100mg/ml pure PG and a 100ml mix with 3mg/ml output shows correct 3ml but 11g on the weight.

**Comment by skydel0** [comment]
Yes there's a Issues. i have to calculate the wight vor the PG and VG parts of the Nikotin Base.
- Fix is done

Please check version v1.29 when it's Online :D
